### PR TITLE
[Merged by Bors] - feat (MeasureTheory.Decoposition.RadonNikodym): `integral_toReal_rnDeriv'`

### PR DESCRIPTION
**Pull request by LorenzoLuccioli**
Add `integral_toReal_rnDeriv'`, an alternative version of `integral_toReal_rnDeriv` for the non absolutely continuous case.

---

- [x] depends on: #15540

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by JasonKYi** [comment]
Thanks!
maintainer merge

File: Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean [hunk 1]
```diff
@@ -390,6 +390,14 @@ lemma integral_toReal_rnDeriv [SigmaFinite μ] [SigmaFinite ν] (hμν : μ ≪
     ∫ x, (μ.rnDeriv ν x).toReal ∂ν = (μ Set.univ).toReal := by
   rw [← integral_univ, setIntegral_toReal_rnDeriv hμν Set.univ]
 
+lemma integral_toReal_rnDeriv' {α : Type*} {m : MeasurableSpace α}
+    {μ : Measure α} {ν : Measure α} [IsFiniteMeasure μ] [SigmaFinite ν] :
```
Comment (RemyDegenne) [inline Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean:394]: Certainly `α` and the measures are already defined in a `variable` line somewhere above?
```suggestion
lemma integral_toReal_rnDeriv' [IsFiniteMeasure μ] [SigmaFinite ν] :
```

File: Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean [hunk 2]
```diff
@@ -390,6 +390,14 @@ lemma integral_toReal_rnDeriv [SigmaFinite μ] [SigmaFinite ν] (hμν : μ ≪
...
+    ∫ (x : α), (μ.rnDeriv ν x).toReal ∂ν
+      = (μ Set.univ).toReal - ((μ.singularPart ν) Set.univ).toReal := by
```
Comment (RemyDegenne) [inline Mathlib/MeasureTheory/Decomposition/RadonNikodym.lean:396]:
```suggestion
    ∫ x, (μ.rnDeriv ν x).toReal ∂ν
      = (μ Set.univ).toReal - (μ.singularPart ν Set.univ).toReal := by
```

**Review by RemyDegenne — APPROVED** [review]
Thanks!
bors d+

**Comment by LorenzoLuccioli** [comment]
Thanks!
bors r+

**Pull request closed without merging**